src/client/async_std.rs

```diff
@@ -196,22 +196,27 @@ fn send<T: DeserializeOwned + Send + 'static>(
 ) -> Response<T> {
     let client = client.clone(); // N.B. Client is send sync;  cloned clients share the same pool.
     Box::pin(async move {
-        let mut response = client.send(request).await?;
-        let status = response.status();
-        let bytes = response.body_bytes().await?;
-        if !status.is_success() {
-            Err(serde_json::from_slice(&bytes)
-                .map(|mut e: ErrorResponse| {
-                    e.error.http_status = status.into();
-                    StripeError::from(e.error)
-                })
-                .unwrap_or_else(StripeError::from))
-        } else {
-            serde_json::from_slice(&bytes).map_err(StripeError::from)
-        }
+        let bytes = send(&client, request).await?;
```

Comment:
Im guessing this is supposed to be `inner_send`?

Reply:
Good catch, thanks! I meant to push this fix a while ago...